charts/am-org-role-mapping-service/values.preview.template.yaml

```diff
@@ -77,7 +77,8 @@ orm:
 
 am-role-assignment-service:
   java:
-    image: 'hmctspublic.azurecr.io/am/role-assignment-service:latest'
+    # TODO remove temp override: image: 'hmctspublic.azurecr.io/am/role-assignment-service:latest'
```

Comment:
Temporary override to be removed when hmcts/role-assignment-service#2541 is released.